packages/desktop-client/src/components/accounts/Balance.tsx

```diff
@@ -42,32 +56,40 @@ function DetailedBalance({ name, balance, isExactBalance = true }) {
   );
 }
 
-function SelectedBalance({ selectedItems, account }) {
+function SelectedBalance({
+  selectedItems,
+  account,
+}: {
+  selectedItems: Set<string>;
+  account: AccountEntity;
+}) {
   const { t } = useTranslation();
 
+  type SelectedBalanceName = `selected-balance-${string}`;
   const name = `selected-balance-${[...selectedItems].join('-')}`;
 
-  const rows = useSheetValue({
-    name,
+  const rows = useSheetValue<'balance', SelectedBalanceName>({
+    name: name as SelectedBalanceName,
     query: q('transactions')
       .filter({
         id: { $oneof: [...selectedItems] },
         parent_id: { $oneof: [...selectedItems] },
       })
       .select('id'),
   });
-  const ids = new Set((rows || []).map(r => r.id));
+  const ids = new Set(Array.isArray(rows) ? rows.map(r => r.id) : []);
```

Comment:
I think this always returns an array

Reply:
`const ids = new Set((rows || []).map(r => r.id));`

complains because rows doesn't always return an array. It can return a number.